sys-utils/libcmt/src/io-mock.c

```diff
@@ -31,6 +31,9 @@ int cmt_io_init(cmt_io_driver_t *_me) {
     }
     cmt_io_driver_mock_t *me = &_me->mock;
 
+    if (me->tx->begin || me->rx->begin) {
```

Comment:
The documentation states to pass an uninitialized `cmt_io_driver_t`. This change affects this behavior and requires it to be zeroed out instead. Please either remove this or update the docs accordingly.

https://github.com/cartesi/machine-emulator-tools/pull/29/files#diff-4cddbbd58ea514448fb003c67bd7bbd304b22d42cbb8a151fcbc776bbacfca48R134